evergreen/run-push.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -o errexit  # Exit the script with error if any of the commands fail
+
+if [ -z "$PACKAGE_VERSION" ]
+then
+  PACKAGE_VERSION=$(git describe --tags)
+  echo Calculated PACKAGE_VERSION value: "$PACKAGE_VERSION"
```

Comment:
In Driver's CI we agreed not to calculate PACKAGE_VERSION for package_push script to avoid accidental pushing incorrect version to nuget.